extra_views/__init__.py

```diff
@@ -10,7 +10,7 @@
 from extra_views.dates import CalendarMonthView
 from extra_views.formsets import FormSetView, InlineFormSetView, ModelFormSetView
 
-__version__ = "0.14.0"
+__version__ = "0.15.0"
```

Comment:
We don't need to perform a release yet as there aren't any actual code changes. Could you revert this part?